src/main/java/com/google/cloud/tools/GCloudErrorException.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.tools;
+
+/**
+ * Signals abnormal gcloud CLI execution.
+ */
+public class GCloudErrorException extends Exception {
+  public GCloudErrorException(int status) {
+    super(String.format("gcloud terminated with an error. Status: %d", status));
```

Comment:
Removed String.format everywhere except in a couple of places where two concats are required.